### DISCUSSION TITLE
Feedback on organization-level security overview

### DISCUSSION THREAD

**Discussion by bewuethr** [community/community#48504] [Code Security]
### Select Topic Area

Product Feedback

### Body

This is about the overhauled tab at `https://github.com/orgs/ORGANIZATION/security/risk`.

- The "team" filter both in the Risk and Coverage views is based on teams having write access to repositories; in an organization where most teams have write access to everything, it's not very useful. In our case, it would be interesting to be able to filter by teams in `CODEOWNERS` instead.
- In the Risk view, it would be interesting to be able to filter by ecosystem.

Combined, these two would let teams answer questions like "which of the repos we own has the most open Go vulnerabilities?", and combined with the [upcoming historical views](https://github.com/github/roadmap/issues/655), you could surface which teams need more support to stay on top of their vulnerabilities.

**Comment by kellyarwine** [comment]
Thanks so much for the feedback @bewuethr.  We appreciate it. Please know we're always listening, pondering and iterating on features based on your feedback.  If you'd like to ~book time~ with me to talk about your feedback, I'm interested.  

(I'll circle back in 7 days and delete my calendar link to avoid spam.)

**Reply by bewuethr** [reply]
Hi Kelly, that URL says it's invalid (same on the security tab itself, by the way).

**Reply by kellyarwine** [reply]
Oops!  Try this: _deleted_

**Comment by CDCR-KevinBaroni** [comment]
This is regarding the new Security tab accessible from the organization. First have to say that you have begun with an admirable start on the new Risk and Coverage visibility and feature-set. Obviously, this is a work-in-progress, but kudos are in order :).  

I represent a large group of developers at the state of California (CDCR). One thing that stands out as a sore spot about all the new very useful Security features is that, unless you click the organization-level Security tab you are unaware that there are any security issues at all. The organizational level Security tab should have some sort of badge indicating that there are security concerns within repos inside the organization. 

A Security badge indicating a total count (depend-a-bot, code scan, secret scan) "may" be impractical as you could have hundreds/thousands of them in an organization with many repos. However, you might want to consider using color coding of the Security tab itself, just as you do with the security warnings such as: Green = no security issues, Yellow = low-to-medium security issues found, Red = high-to-critical security issues found.  Something along those lines would be of great value to organization owners and helps us to be more efficient at our jobs.
